src/individual_metrics.py

```diff
@@ -0,0 +1,118 @@
+"""
+This module is intended to run after all data has been collected and added.
+
+to the .json file. The purpose is to create a dictionary where the username is.
+
+the key and other calculated metrics are the values.
+"""
+from __future__ import division
+import json_handler
+import data_collection
+
+# from pydriller.domain.commit import ModificationType
+
+
+def get_commit_average(lines, commits):
+    """Find average lines modified per commit."""
+    # Loop through the dictionary and calculate the average lines per commits
+    # formula for average: (added + deleted)/number_of_commits
+    if commits != 0:
+        average = lines / commits
+        return (int)(average)
+    return 0
+
+
+def parse_for_type(name):
+    """Parse through file name and returns its format."""
+    if "." in name:
+        dot_index = name.find(".")
+        return name[dot_index:]
+    return name
```

Comment:
There are some [built-in functions](https://realpython.com/working-with-files-in-python/) in `os` that can just return the file names.

Reply:
Great idea! I just added an os function into `get_file_formats()`.